akskafka-databricks-cosmosdb/create-solution.sh

```diff
@@ -170,18 +170,18 @@ echo "***** [T] Starting up TEST clients"
 
     RUN=`echo $STEPS | grep T -o || true`
     if [ ! -z "$RUN" ]; then
-        source ../components/azure-kubernetes-services/get-aks-kafka-brokers.sh
+        source ../components/azure-kubernetes-service/get-aks-kafka-brokers.sh
         source ../simulator/run-generator-kafka.sh
     fi
 echo
 
 echo "***** [M] Starting METRICS reporting"
 
-    RUN=`echo $STEPS | grep M -o || true`
-    if [ ! -z "$RUN" ]; then
-        # TODO: Create aks report-throughput.sh
-        # source ../components/azure-event-hubs/report-throughput.sh
-    fi
+    # RUN=`echo $STEPS | grep M -o || true`
```

Comment:
Will we use that sometime? If we are not, can we get rid of it?

Reply:
This will be the monitoring script we still need to create